sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -942,7 +938,7 @@ export class MessageSession extends LinkEntity {
     this.isReceivingMessages = true;
 
     return new Promise<ServiceBusMessage[]>((resolve, reject) => {
-      let totalWaitTimer: NodeJS.Timer | undefined;;
+      let totalWaitTimer: any;
```

Comment:
What prompted this change?

Reply:
the build was failing, this should have already been fixed on master.
Looks like there might have been a bad merge somewhere, and might explain the browser test failures as well

Reply:
I can confirm that this is not an issue in master